extension/test/data_test.ts

```diff
@@ -69,4 +69,166 @@ describe('data.ts', function () {
       ).to.include.something.like({ entry: /^ＤＶＤ-ＲＯＭ .*/ });
     });
   });
+
+  describe('makeText', function () {
+    describe('with a word dict entry', function () {
+      // wordDictEntry was generated with: rcxDict.wordSearch('あ', /* doNames= */ false)
```

Comment:
I'm not sure if this comment is helpful. If it's not helpful, I can remove it.

Reply:
I think removing it makes sense to me. The structure of the entry object is independent and over time it's possible that the output of `wordSearch` and these tests will diverge making the comment a lie.